index.js

```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (c) 2014, Yahoo Inc. All rights reserved.
```

Comment:
@ericf Yes. I was following the guidelines in the Jive OSWG place. I looked at the latest commit dates of the files.